### PR TITLE
Core should check apache auth

### PR DESCRIPTION
**Pull request by tomneedham**
This fixes Apache based logins in 10.0.8.

### CODE REVIEW

**Comment by tomneedham** [comment]
Fixes https://github.com/owncloud/core/issues/31052

**Comment by tomneedham** [comment]
Looks like its the `IApacheBackend` that needs the caching fix for this - not core.

**Comment by tomneedham** [comment]
Tested in the wild ✅

**Comment by PVince81** [comment]
sounds too risky for now, moving to "planned"

File: core/Controller/LoginController.php [hunk 1]
```diff
@@ -26,6 +26,7 @@
 namespace OC\Core\Controller;
 
 use OC\Authentication\TwoFactorAuth\Manager;
+use OC\User\LoginException;
```
Comment (tomneedham) [inline core/Controller/LoginController.php]: remove this

**Review by butonic — APPROVED** [review]

**Comment by PVince81** [comment]
ok, so we do want it because it fixes a regression with shib

**Pull request merged**